packages/react/index.js

```diff
@@ -59,6 +59,7 @@ export {
   createMutableSource,
   Fragment,
   Profiler,
+  DebugTraceMode,
```

Comment:
Wouldn't this add it to public API? Let's make sure it stays FB-only or at least fully behind a flag.

Also maybe add unstable_

Reply:
It's only exposed in the experimental (and FB) indexes. This is the main index, not the `stable` one.